### DISCUSSION TITLE
Update Message Reaction

### DISCUSSION THREAD

**Discussion by benedettosiddi** [python-telegram-bot/python-telegram-bot#3867] [Feature Suggestions]
Is it possible to get the updated message reaction as long as the message id?

**Comment by Bibo-Joshi** [comment]
The bot API doesn't include any functionality related to message reactions

**Comment by sabrysm** [comment]
It should be now possible!

**Comment by Raphmore27** [comment]
Please I used from telegram import Update and it's showing an error message that no module...
